core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -45,9 +53,28 @@ export class ContainerListComponent implements OnInit, OnDestroy {
             .subscribe(([isEnterPrise, hasEnvironments]: [boolean, boolean]) => {
                 this.isEnterPrise = isEnterPrise;
                 this.hasEnvironments = hasEnvironments;
-                this.tableColumns = this.setContainerColumns();
-                this.containerBulkActions = this.setContainerBulkActions();
+                this.store.updateTableColumns(this.setContainerColumns());
+                this.store.updateContainerBulkActions(this.setContainerBulkActions());
             });
+
+        this.vm$
+            .pipe(takeUntil(this.destroy$))
+            .subscribe(
+                ({
+                    containerBulkActions,
+                    selectedContainers,
+                    addToBundleIdentifier,
+                    actionHeaderOptions,
+                    tableColumns
+                }: DotContainerListState) => {
+                    this.containerBulkActions = containerBulkActions;
+                    this.selectedContainers = selectedContainers;
+                    this.addToBundleIdentifier = addToBundleIdentifier;
+                    this.actionHeaderOptions = actionHeaderOptions;
+                    this.tableColumns = tableColumns;
```

Comment:
This should be a [select](https://ngrx.io/guide/component-store/read#select-method) in the component store that you use directly in your template.

Probably just one select that includes `actionHeaderOptions `, `containerBulkActions` and this one.